### PR TITLE
Privacy: updated sign-up and privacy UI

### PR DESCRIPTION
**Pull request by breville**
#### sign up:
screenshot 2018-02-15 17 33 24

#### privacy desktop:
screenshot 2018-02-15 17 33 40

#### privacy mobile:
screenshot 2018-02-15 17 33 55

### CODE REVIEW

**Comment by tanyaparker** [comment]
Looks great!! So we're just waiting to hear back from Common Sense for a new link to the evaluation and then we can also put a Common Sense badge on the privacy policy page.

@poorvasingal what do you think about the quote that we use from the evaluation? Seems kinda repetitive to the first principle. Perhaps when we get the evaluation again we can scan for a better one? Or do we have an agreement with them?

File: dashboard/app/views/devise/registrations/new.html.haml [hunk 1]
```diff
@@ -117,11 +117,7 @@
     .span4
       .devise_links
         = render "devise/shared/links"
-
-    %div{style: "clear:both"}
-    %br/
-    %br/
-    %br/
+        = render "devise/shared/policylogos"
```
Comment (tanyaparker) [inline dashboard/app/views/devise/registrations/new.html.haml:120]: Confirming that this is EN only?

**Comment by poorvasingal** [comment]
@tanyaparker - Yeah I had the same thought about the quote. I don't think I originally realized that they just repeated one of our principles. Yes, let's wait until we hear back from them about the link so we can find a better quote. Looks good to me otherwise. Thanks!

**Review by Erin007 — APPROVED** [review]

**Comment by tanyaparker** [comment]
[Privacy policy logos on our signup page and privacy policy page](https://trello.com/c/dBJiHYJg/94-privacy-policy-logos-on-our-signup-page-and-privacy-policy-page)

**Pull request closed without merging**